_docker/drupal/drupal-filesystem/README.md

```diff
@@ -137,3 +137,130 @@ To view diffs in a content node, navigate to the node that you would like to edi
 4. Select compare and observe changes.
 
 
+## Making changes to the Drupal database
+
+There may be occasions when you need/want to make changes to the Drupal database through a mechanism other than Drupals
+interface. In this scenario, we use [Phinx](https://phinx.org) to create and apply your database changes. Changes are not
+applied to the databases by hand.
+
+To use phinx in your local development environment, you will need to run `composer install` from the `_docker/drupal/drupal-filesystem`
+directory. This will install the phinx binary into `_docker/drupal/drupal-filesystem/vendor/bin/phinx`
+
+### Writing database changes with Phinx
+
+Changes to the database managed by Phinx are called migrations. Each time you want to make a change to the database, you
+ should create a new migration. As part of the Drupal boot sequence, we instruct Phinx to apply all migrations within
+ the `_docker/drupal/drupal-filesystem/db-migrations` directory against the database for the current environment.
+
+To create a new migration, do the following:
+
+```bash
+cd _docker/drupal/drupal-filesystem
+vendor/bin/phinx create <Name_Of_Your_Migration>
+```
+
+This will generate a PHP class in the `_docker/drupal/drupal-filesystem/db-migrations` directory. The PHP class is an
+abstract Phinx class that provides basic methods and an API to write the changes to want to make to the database in
+PHP code. Phinx has good documentation [here](http://docs.phinx.org/en/latest/migrations.html) on how to use the PHP 
+migration classes to perform changes to the database.
+
+Ideally your migration should support rolling back your change. Phinx can automatically rollback some changes 
+(see the documentation for which changes it can automatically rollback). However if Phinx is not capable of automatically
+rolling back your change, then ideally you should provide implementations of the `up` method to make your change and the
+`down` method to roll it back.
+
+You should commit your change as part of your pull-request so that it can be reviewed by other members of the team. 
+
+Phinx will only ever apply a change to a database once. It will apply the changes in the timestamp order of their class
+name. Once a change has been applied, Phinx will write an entry into the `phinxlog` table in the database. 
+
+### Manually testing a migration in your local development environment
+
+To manually test your migration, you will first need to have the database running on your local machine. The easiest way
+to do this is to use the following:
+
+```bash
+cd _docker/environments/drupal-dev
+docker pull redhatdeveloper/drupal-data-lite:latest
+docker-compose up -d drupalmysql
+```
+
+Once the database has booted, you should verify that you can connect to it using the following:
+
+```bash
+mysql -h 0.0.0.0 -u root -p
```

Comment:
Not sure if it really matters either. If we've used the drupal user elsewhere in the documentation though, we should stay consistent.